viv_utils/emulator_drivers.py

```diff
@@ -152,46 +219,41 @@ def doHook(self, pc, op):
         for mon in self._monitors:
             try:
                 r = mon.apicall(self, op, pc, api, argv)
-                if r is not None:
-                    # take the first result
-                    # not ideal, but works in the common case
-                    logger.debug("driver: monitor hook handled call: %s", callname)
-                    return True
-            except StopEmulation:
-                raise
```

Comment:
will this still be available to higher-level callers?

Reply:
i messed up a merge and forgot this, sorry

Reply:
re-added in 5df84599ee18f4ec7deddf43f3b8926d01a951b8